uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
+1. From the command prompt, start the Windows Application Driver service:
+```cmd
+"C:\Program Files\Windows Application Driver\WinAppDriver.exe"
```

Comment:
Nit: Maybe I'm thinking PS, but don't you need a `&` at the beginning to execute something like this?

Reply:
`&` prefix syntax is required for running the service in Powershell. In `cmd` we can simply start the process by submitting the absolute path to the executable